sway/commands/force_focus_wrapping.c

```diff
@@ -1,20 +1,12 @@
-#include <strings.h>
+#include "log.h"
 #include "sway/commands.h"
-#include "sway/config.h"
-#include "util.h"
 
 struct cmd_results *cmd_force_focus_wrapping(int argc, char **argv) {
-	struct cmd_results *error =
-		checkarg(argc, "force_focus_wrapping", EXPECTED_EQUAL_TO, 1);
-	if (error) {
-		return error;
+	sway_log(SWAY_INFO, "Warning: force_focus_wrapping is deprecated. "
+		"Use focus_wrapping instead.");
+	if (config->reading) {
+		config_add_swaynag_warning("force_focus_wrapping is deprecated. "
+			"Use focus_wrapping instead.");
 	}
-
-	if (parse_boolean(argv[0], config->focus_wrapping == WRAP_FORCE)) {
```

Comment:
Even with the deprecation warning, this block should be retained. The arguments for `force_focus_wrapping` are different then`focus_wrapping`.